hack/make/release-deb

```diff
@@ -14,6 +14,7 @@ set -e
 #
 # ... and so on and so forth for the builds created by hack/make/build-deb
 
+source "$(dirname "$BASH_SOURCE")/.integration-daemon-start"
```

Comment:
Maybe instead of those two lines we can do:

```
$ echo $(go env GOHOSTOS)/$(go env GOHOSTARCH)
linux/amd64
```

I think it should be equivalent.